data/hazards.txt

```diff
@@ -124,3 +124,20 @@ hazard "Rulei Gravity Field"
 
 effect "gravity well"
 	"sound" "gravity well"
+
+
+hazard "Core Sheltered Heat"
+	"duration" 1
+	"strength" 0.65 0.85
+	"range" 50000
+	weapon
+		"burn damage" 0.06
+		"relative heat damage" 0.0012
+#replace 0.0006 rel. heat damage with 0.00006 rel. burn damage if/when implemented.
+
+hazard "Core Shadow Passing"
+	"duration" 10 360
+	"strength" 1 3
+	"range" 50000
+	weapon
+		"relative heat damage" -0.002
```

Comment:
Yes, that is intentional. It needs to add a significant cooling off amount to achieve the desired effect. It is not used on its own.

Reply:
The actual hazard implementation in a system should never be set up to have that much of an effect, but the hazard definition needs to be capable of this or it actually just doesn't work.

Reply:
I'd have to agree with Azure on this one. We shouldn't be doing any sort of system-wide hazards like that in human space, at least space that is accessible to the player early on. It seems much better reserved for uninhabited core + xapleaux systems rather than creeping into humans'.

Reply:
It's only applied *very gently* to the one system (Polaris or Sheratan, I forget which) where the planet description calls out how much brighter and closer to the core it is. It's not implemented at a level that's remotely capable of soft-locking anyone in human space.

It is however used for the uninhabited systems beyond that that are included as part of the overall core system package.
Which again, is not meant to be set at a level capable of softlocking anyone who's ship outfitting isn't already running way way too hot to be doing anything safely.

Reply:
It is not in human space. Look at the map file.

Reply:
Human space having a gentler version of more dangerous hazards further out in space is a good way to introduce the player to the possibility that these hazards could possibly even exist out there in space. It depends on the actual geography of the galaxy, but I would want a fairly gradual change from no heat to significantly dangerous bouts of heat, and it should occupy multiple systems, especially over the entirety of at least one route between two inhabited systems. I think that would inoculate the player to the notion of "significant heat can come from the space you're sitting in, too."